components/infieldprogresscircle/index.css

```diff
@@ -0,0 +1,36 @@
+/*!
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-InfieldProgressCircle {
+	--mod-progress-circle-thickness: var(--mod-progress-cirlce-stroke-width, 2px);
+	--mod-progress-circle-size: var(--spectrum-in-field-progress-circle-size-100);
+}
+
+.spectrum-InfieldProgressCircle--sizeS {
+	--mod-progress-circle-size: var(--spectrum-in-field-progress-circle-size-75);
+}
+
+.spectrum-InfieldProgressCircle--sizeL {
+	--mod-progress-circle-size: var(--spectrum-in-field-progress-circle-size-200);
+}
+
+.spectrum-InfieldProgressCircle--sizeXL {
+	--mod-progress-circle-size: var(--spectrum-in-field-progress-circle-size-300);
+}
+
+.spectrum-InfieldProgressCircle {
+	padding-block: var(--spectrum-in-field-progress-circle-edge-to-fill);
```

Comment:
I believe the preference is not to use the tokens directly in style property definitions like this. Instead of `padding-block: var(--spectrum-in-field-progress-circle-edge-to-fill);`, could you make another custom property for this, above the mods, and then use the custom prop to define `padding-block`? 

```
.spectrum-InfieldProgressCircle {
	--spectrum-infieldprogresscircle-padding-block: var(--spectrum-in-field-progress-circle-edge-to-fill);
// mods are here
}
... // t-shirt size stuff you already have

.spectrum-InfieldProgressCircle {
	padding-block: var(--spectrum-infieldprogresscircle-padding-block);
...
}
```